Update JSTL TCK User Guide for Tags 3.0

volosied opened the pull request:
~ Ready for Review  ~
- Updated attributes to reference Tags version 3.0 and Glassfish 7.0
- Updated dates & references to EE 10. 
- Removed COMPAT documentation as that's no longer needed (since the tests were updated to use the default Java 11 locales. See https://github.com/eclipse-ee4j/jakartaee-tck/issues/631#issuecomment-799764345) 

CC @alwin-joseph @anajosep @arjantijms @cesarhernandezgt @dblevins @m0mus @edbratt @gurunrao @jansupol @jgallimore @kazumura @kwsutter @LanceAndersen @bhatpmk @RohitKumarJain @shighbar @gthoman @brideck @scottmarlow

scottmarlow commented:
Should also update the user_guides/jstl/src/main/jbake/content/install-server.inc which also still references `EE 8` and is included from install.adoc.  The same for rebuild.inc in the same folder.

volosied commented:
Thanks for spotting that. I updated all  `EE 8` references to `EE 10` in these 3 files. 

        modified:   src/main/jbake/content/config.inc
        modified:   src/main/jbake/content/install-server.inc
        modified:   src/main/jbake/content/rebuild.inc

volosied commented:
@scottmarlow  Sure, let me take a look at this again. 

One question, should I specify only JDK 11  or should JDK 17  also be mention for the Java SEversion?  I saw the TCK is run for both versions.

scottmarlow commented:
@volosied Please do mention both JDK 11 and JDK 17 as the [EE 10 release plan ](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)calls for both to be supported:

>
>TCK Source Level
>
>The TCKs will be compiled at the Java SE 11 level. In order to allow for runtimes beyond Java SE 11, the TCK will also need to be (successfully) executed with Java SE 17.

volosied commented:
Section 4.1 Step 2 says: 
`Copy <TS_HOME>/bin/ts.jte.jdk11 as <TS_HOME>/bin/ts.jte if JAVA_HOME is Java SE 11. Edit your <TS_HOME>/bin/ts.jte file and set the following environment variables:` 

Should this be removed/edited since java 11 is the minimum for the TCK now?

volosied commented:
@pnicolucci   @arjantijms  Let me know if this looks good (or if any more changes are needed). Thanks

volosied commented:
@volosied  I believe you're right -- I marked it as ready for review.  Thank you.

gurunrao commented on user_guides/jstl/src/main/jbake/content/attributes.conf:
```diff
@@ -1,12 +1,12 @@
-:CopyrightDates: 2017, 2021
-:TechnologyRI: Eclipse GlassFish 6.1
+:TechnologyVersion: 3.0
+:ReleaseDate: March 2022
```
We can update to April

volosied replied:
I updated it to May instead  as that month is more likely at this point.

gurunrao commented on user_guides/jstl/src/main/jbake/content/rebuild.inc:
```diff
@@ -23,7 +23,7 @@ published in a Java SE environment are located under
 
 The {TechnologyShortName} TCK comes with prebuilt test WAR files for
-deployment on Java EE 8 RI , which provides a Servlet–compliant Web
+deployment on Java EE 10 RI , which provides a Servlet–compliant Web
```
Java EE to Jakarta EE

volosied replied:
Fixed

scottmarlow commented:
Started https://ci.eclipse.org/jakartaee-tck/job/10/job/eftl-standalonetck-build-run-100/106 to build the JSTL TCK
